pkg/upgrade/upgrader/helper.go

```diff
@@ -111,7 +111,8 @@ func getContainerName(d *appsv1.Deployment) (string, error) {
 func getBaseImage(deployObj *appsv1.Deployment, name string) (string, error) {
 	for _, con := range deployObj.Spec.Template.Spec.Containers {
 		if con.Name == name {
-			baseImage := strings.Split(con.Image, ":")[0]
+			lastIndex := strings.LastIndex(con.Image, ":")
```

Comment:
colon ':' is not a mandatory requirement for image. How are we handling it?
Can we add a test case where image does not have any `:`

Reply:
Not having a `:` would implicitly mean `latest` ? Maybe that should not be supported - just raise an error?

Reply:
Thanks @davidkarlsen. Added error message for no image tag.